src/transformers/configuration_mpnet.py

```diff
@@ -0,0 +1,77 @@
+import logging
+
+from .configuration_utils import PretrainedConfig
+from .configuration_bert import BertConfig
+
+
+logger = logging.getLogger(__name__)
+
+
+MPNET_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "mpnet-base": "",
+}
+
+
+class MPNetConfig(PretrainedConfig):
+    r"""
+        :class:`~transformers.MPNetConfig` is the configuration class to store the configuration of a
```

Comment:
This docstring should be adapted to the new standard (copy from any existing config file, like `configuration_bert.py`).